app/src/main/java/com/example/android/uamp/viewmodels/NowPlayingFragmentViewModel.kt

```diff
@@ -177,6 +177,16 @@ class NowPlayingFragmentViewModel(
                 NowPlayingMetadata.timestampToMSS(app, mediaMetadata.duration)
             )
             this.mediaMetadata.postValue(nowPlayingMetadata)
+
```

Comment:
Placing the code to store the recently played media id here won't work if the UI has been closed. It should be done inside the `MusicService`